frontend/src/pages/account/Settings.vue

```diff
@@ -1,12 +1,26 @@
 <template>
     <ff-loading v-if="loading" message="" />
     <form v-else class="space-y-6">
```

Comment:
can we add an `@submit` here please? Even if it's just an `@submit.enter.prevent=""` to disable the behaviour.

Reply:
Added